lte/gateway/docker/services/python/Dockerfile

```diff
@@ -170,7 +170,9 @@ COPY --from=builder /build /build
 COPY --from=builder /var/tmp/python3-aioeventlet*.deb /var/tmp/
 
 # The Python scripts in /build/bin are broken inside the container
-RUN rm -f /build/bin/dhcp_helper_cli.py
+# Remove all Python scripts already exist in /usr/local/bin
+WORKDIR /usr/local/bin
+RUN find ./*.py -exec rm -rf /build/bin/{} \;
```

Comment:
nit: the "./" at the start of `find` is not necessary, and it will be added to the `rm` command as well, i.e. the file path will be `/build/bin/./*.py`. It doesn't change the functionality, though, so it should be fine as it is.